stackrox/stackrox
# ROX-20353: Reconcile image integrations

## Description

* Depends on: #8310 

Adding `Image Integrations` to the `reconciliationStore` and reconcile them upon connection with Central.

## Checklist

If any of these don't apply, please comment below.

## Testing Performed

   * Deploy ACS
   * Create a secret that will trigger a `ImageIntegration` event: `kubectl create secret docker-registry test --docker-username=test --docker-password=1234 --docker-server=registry.test:5000`
   * Scale down Sensor
   * Delete the secret
   * Scale up Sensor
   * The secret should not appear in the `image_integrations` table

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.
